wordle-app/pages/WordOfTheDayGame.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+  <v-container>
+    <v-progress-linear
```

Comment:
I would suggest altering your `game.ts` file that will take in a boolean to see if its a daily game or not daily game. Then you can componentize the all of this into something like `game.vue` with a prop of type. That way both pages can just use a `Game` component and then have one have `isDaily=true` and `isDaily=false` That way it more effectively reuses components

Reply:
This is a great suggestion. Thank you